src/keycloak/chart/dashboards/uds-keycloak-notifications.json

```
@@ -0,0 +1,650 @@
+                                "value": 80
+                            }
+                        ]
+                    }
+                },
+                "overrides": []
+            },
+            "gridPos": {
+                "h": 8,
+                "w": 24,
+                "x": 0,
+                "y": 25
+            },
+            "id": 7,
+            "options": {
+                "cellHeight": "sm",
+                "showHeader": true
+            },
+            "pluginVersion": "12.2.0",
+            "targets": [
+                {
+                    "datasource": {
+                        "type": "loki",
+                        "uid": "P8E80F9AEF21F6940"
+                    },
+                    "direction": "backward",
+                    "editorMode": "code",
+                    "expr": "{app=\"keycloak\", namespace=\"keycloak\"} | json | loggerName=\"uds.keycloak.plugin.eventListeners.JSONLogEventListenerProvider\" | eventType=\"ADMIN\" | resourceType=~\"USER|GROUP_MEMBERSHIP\"",
+                    "queryType": "range",
+                    "refId": "A"
+                }
+            ],
+            "title": "User Modification Logs",
+            "transformations": [
+                {
+                    "id": "extractFields",
+                    "options": {
+                        "delimiter": ",",
+                        "replace": true,
+                        "source": "Line"
+                    }
+                },
+                {
+                    "id": "organize",
+                    "options": {
+                        "excludeByName": {
+                            "eventType": true,
+                            "id": true,
+                            "loggerName": true,
+                            "realmId": true,
+                            "resourceTypeAsString": true,
+                            "time": true
+                        },
+                        "includeByName": {},
+                        "indexByName": {
+                            "authDetails": 11,
+                            "details": 13,
+                            "error": 12,
+                            "eventType": 2,
+                            "id": 3,
+                            "loggerName": 1,
+                            "operationType": 8,
+                            "realmId": 5,
+                            "realmName": 6,
+                            "representation": 10,
+                            "resourcePath": 9,
+                            "resourceType": 7,
+                            "resourceTypeAsString": 14,
+                            "time": 4,
+                            "timestamp": 0
+                        },
+                        "renameByName": {}
+                    }
+                }
+            ],
+            "type": "table"
+        }
+    ],
+    "preload": false,
+    "refresh": "",
+    "schemaVersion": 42,
+    "tags": [],
```

Might want to consider adding some tags. For one other custom dashboard we included `uds` and `istio`, so maybe this?

```suggestion
    "tags": [
        "keycloak",
        "uds"
    ],
```